### PR TITLE
User Story 1 - React/TS/Jest/Tailwind Boilerplate

### PR DESCRIPTION
**Pull request by Durbatuluk1701**
Add initial boilerplate code for a React/TypeScript/Tailwind project with Jest.

Note: I am not sure how well tailwind is configured yet, but I think building the templates and stuff should be a separate user story. Would be nice to get this merged in soon so we can build off it

### CODE REVIEW

**Review by Durbatuluk1701 — COMMENTED** [review]
Including package-lock.json seems a bit excessive, what are others thoughts?

**Comment by maxxxxxdlp** [comment]
Yes, that file is a bit big, but it should be checked in.

See https://stackoverflow.com/questions/44206782/do-i-commit-the-package-lock-json-file-created-by-npm-5

**Comment by maxxxxxdlp** [comment]
Also, thanks for taking a head start on this!
I meant to get to this yesterday but didn't get a chance.

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/web-dev/node_modules
```
Comment (maxxxxxdlp) [inline .gitignore:4]: Can you please rename the root folder from `web-dev` to `src` or `lib`?
I believe those are more common names

**Review by maxxxxxdlp — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (Durbatuluk1701) [reply .gitignore]: Sounds good, think I changed it everywhere

**Review by maxxxxxdlp — APPROVED** [review]

**Comment by maxxxxxdlp** [comment]
Fixed https://github.com/maxxxxxdlp/calendar-stats/pull/1

**Comment by maxxxxxdlp** [comment]
Fixed https://github.com/maxxxxxdlp/calendar-stats/issues/3

**Pull request merged**